SpatialGDK/SpatialGDK.uplugin

```diff
@@ -74,6 +74,15 @@
         "Mac"
       ]
     },
+    {
```

Comment:
I'm not sure how this is creeping up, not the first time. Also happened with Vicky.